src/lua/compat.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#include <lua/compat.h>
+
+static const char escaped_slash[] = "\\/";
+
+int
+msgpuck_esc_slash_toggle(struct lua_State *L)
```

Comment:
The one and only purpose of `compat.{c,h}` is to contain LuaC functions for the options. Lua postaction functions are stored in `compat.lua`, so one can expect that LuaC postaction functions are stored in LuaC part of compat. 

> Or put function msgpuck_esc_slash_toggle to source code of msgpuck like you did with json_esc_slash_toggle.

It would require a patch to msgpuck tarantool fork, and would unnecessary complicate the patchset.

Reply:
Summarized my opinion in https://github.com/tarantool/tarantool/pull/6910#issuecomment-1260924239.